lib/enum/enum.rb

```diff
@@ -29,6 +28,20 @@ def delete_all
         super
       end
 
+      private
+
+      def define_enum_methods
+        return if @enum_accessors.blank?
+
+        @records&.each do |record|
+          enum_vallue = @enum_accessors.map { |name| record.attributes[name] }.join("_").downcase.gsub(/\W+/, '_').gsub(/^_|_$/, '')
```

Comment:
Please name this variable `enum_value` which is the conventional spelling.

Reply:
Fix: https://github.com/active-hash/active_hash/pull/321/commits/9e2fc62b37e5f108ed06ba2e7cd25fed40df7fb5